beancount/core/account.py

```diff
@@ -41,9 +41,9 @@ def is_valid(string):
     This does not check for the root account types, just the general syntax.
 
     Args:
-      string: A string, to be checked for account name pattern.
+      string (str): To be checked for account name pattern
```

Comment:
I prefer not to adopt this style for documenting types, especially as it would require changing the entire set of docstrings to be consistent. In v3 we'll be using pytype everywhere and that might change too.
Can you remove this from the PR?

Reply:
done